apps/cowswap-frontend/src/api/coingecko/index.ts

```diff
@@ -1,135 +0,0 @@
-import { SupportedChainId as ChainId } from '@cowprotocol/cow-sdk'
-
-import useSWR from 'swr'
-import { PriceInformation } from 'types'
-
-import { SWR_OPTIONS } from 'legacy/constants'
-
-import { fetchWithRateLimit } from 'common/utils/fetch'
-
-function getApiUrl(): string {
-  // it's all the same base url
-  return 'https://api.coingecko.com/api'
-}
-
-// https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses=0x33e18a092a93ff21ad04746c7da12e35d34dc7c4&vs_currencies=usd
-// Defaults
-const API_NAME = 'Coingecko'
-const ENABLED = process.env.REACT_APP_PRICE_FEED_COINGECKO_ENABLED !== 'false'
```

Comment:
Has this been removed from everywhere else, like .env files and so on?

Reply:
Thank you! Cleaned up env files